src/makeCalculator.js

```diff
@@ -37,7 +37,44 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    DEFAULT_VALUE: 0,
+    result: 0,
+
+    operate(
+      modifyResult = N => N ?? this.result,
+      n = this.DEFAULT_VALUE,
+    ) {
+      // add context to the callback function and call it
+      modifyResult.bind(this)(n);
+
+      return this;
+    },
+
+    add(n) {
```

Comment:
use more descriptive name for function argument

Reply:
I don't think it's really necessary, we always take a number as a parameter
So what's the best way to name a variable containing a random number

Reply:
I think better to avoid using one-letter variable naming(It could be confusing for another developer, who reads your code)

So let's replace `n` with `number`, maybe a little bit, but it will improve readability of your code